modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -248,7 +248,7 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
     };
 
     // Set width/height properties
-    if (wasResizeStarted) {
+    if (!Rtc.isRtc(editor) && wasResizeStarted) {
```

Comment:
I'm not sure this can actually be done thinking more, as the current expectation is that by the time the `ObjectResized` event has fired, then the DOM would have been updated, which then allows other logic to run. This breaks that, as the assuming you are using that event, then others also using it may get it called without the content being changed.

This would almost definitely break the table plugin logic, as it definitely needs the table to have been resized when the `ObjectResized` event is fired.

Reply:
Maybe we need to consider adding a `BeforeObjectResized` event?